### ISSUE TITLE
How to restore weights of model from zoo as trainable or how to train custom model with object detection script

### ISSUE THREAD

**Issue by VladBlat** [tensorflow/tensorflow#32690]
Hello
I'm trying to add some new outs to ssd mobilenet v2 taken from model zoo. How to restore graph and variables data to train my additional layers and fine tune mobilenet for better future extracting.</br>
In my tries i used tf.Saver and simple import_meta_graph and restore.</br>
Or push me to right way with using TFObjectDetection API for this task

**Comment by oanush** [comment]
@VladBlat ,
Please refer the links [1](https://www.tensorflow.org/guide/saved_model) and [2](https://www.tensorflow.org/beta/guide/keras/saving_and_serializing) and let us know if it helps,thanks!

**Comment by tensorflowbutler** [comment]
It has been 15 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by VladBlat** [comment]
Hello, I didn't find some useful info about this issue at links which u
gave. Restore variables as trainable possible using object detection Api. U
can close topic. Good luck!

вт, 8 окт. 2019 г., 15:38 Alfred Sorten Wolf <notifications@github.com>:

**Comment by oanush** [comment]
Closing since the issue is resolved.Thanks!

**Comment by oanush** [comment]
Closing since the issue is resolved.Thanks!
